voip_oca/models/res_partner.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2025 Dixmit
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+from odoo.osv import expression
+
+
+class VoipOcaCall(models.Model):
+    _inherit = "res.partner"
+
+    def format_partner(self):
+        return {
+            "id": self.id,
+            "type": "partner",
+            "displayName": self.display_name,
+            "email": self.email,
+            "landlineNumber": self.phone,
+            "mobileNumber": self.mobile,
+            "name": self.name,
+        }
+
+    @api.model
+    def get_voip_contacts(self, _search, offset, limit):
```

Comment:
As I see you are doing some kind of "API", please prefix all the methods with `voip` / `_voip` instead:

```suggestion
    def voip_get_contacts(self, _search, offset, limit):
```